tests/x11/shutdown.pm

```diff
@@ -78,7 +81,20 @@ sub run() {
         power('off');
     }
 
-    assert_shutdown;
+    if (check_var('BACKEND', 's390x')) {
+        # make sure SUT shut down correctly
+        console('x3270')->expect_3270(
+            output_delim => qr/.*SIGP stop.*/,
+            timeout      => 30
+        );
+
+        # deactivate consoles
+        do_stop_vm;
+        assert_shutdown;
```

Comment:
as in both if and else assert_shutdown is the last function, move it completely out of the if